map/src/menu/MainMenu.js

```diff
@@ -65,6 +66,14 @@ export default function MainMenu({
     };
 
     const items = [
+        {
+            name: 'Configure Map',
+            icon: ConfigureMapIcon,
+            component: <ConfigureMap />,
+            type: null,
+            show: true,
+            id: 'se-show-menu-configuremap',
+        },
```

Comment:
Please check behavior of this menu in case of REACT_APP_DEVEL_FEATURES='no' (prod) && user logged out. Also check the same for "Configure" button on the map.